packages/site/src/components/ComponentLinks.tsx

```diff
@@ -52,72 +52,87 @@ export const ComponentLinks = ({
   if (isMinimal) return null;
 
   return (
-    <Content>
-      <Box padding="base" direction="column">
-        <Content spacing={"larger"}>
+    <aside
```

Comment:
pretty sure this is the correct semantic element - it could also potentially be a nav